frontend/src/concepts/k8s/ToggleResourceName.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Checkbox, Popover, Icon, Split, SplitItem } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+type ToggleResourceNameProps = {
+  showHiddenResourceName: boolean | undefined;
+  setShowHiddenResourceName: (checked: boolean) => void;
+};
+const ToggleResourceName: React.FC<ToggleResourceNameProps> = ({
+  showHiddenResourceName,
+  setShowHiddenResourceName,
+}) => (
+  <Split hasGutter>
+    <SplitItem>
+      <Checkbox
+        label="Show advanced settings"
```

Comment:
is this expected to be checked by default? It's checked for me by default.

Reply:
@manaswinidas yes, this is expected behavior the checkbox is toggled based on `showK8sName` props